subjects/mobile-dev/hacker-news/audit/README.md

```diff
@@ -1,17 +1,29 @@
 #### Functional
 
-> In order to run and hot reload app either on emulator or device, follow the [instructions](https://docs.flutter.dev/get-started/test-drive?tab=androidstudio#run-the-app)
+###### Does the app show a list of all fetched posts on the main screen?
 
-###### Does app shows all fetched posts?
+###### When you tap on a post, does it open the content in a `webview`?
```

Comment:
Little detail - in the link `WebView` is the format used for the word

Reply:
done